convert/README.md

```diff
@@ -65,32 +69,26 @@ autoscaling group nodes and bastion node will be provisioned with the new key.
 
 ### Get AWS resources to import
 
-Then, we'll use `convert.py` to generate the Terraform import commands to import our AWS resources:
-
-    ./convert.py get-imports --region us-east-1 --stack-name mystack > terraform/imports.sh
-
-### Initialize and import Terraform resources
-
-Change to the `terraform/` directory, and run the following two commands:
+Then, we'll use `convert.py` to generate the Terraform import blocks to import our AWS resources:
 
-    terraform init
-    bash imports.sh
+    ./convert.py get-imports --region us-east-1 --stack-name mystack > terraform/imports.tf
```

Comment:
Although I'd be a tiny bit sad at losing the simplicity of just printing to stdout, I wonder if, now that everything is just terraform anyway, it would make sense to combine the `create-tfvars` and `get-imports` steps, and just have them write directly to files in some specified directory (that probably defaults to `terraform/` so you don't have to think about it 95% of th etime).

Reply:
I am of the same mind, even debated with `run the following commands to populate all the vars` but I landed on the same conclusion of just having the convert do this.